config/cspo/clusterstack.yaml

```diff
@@ -1,3 +1,16 @@
+apiVersion: infrastructure.clusterstack.x-k8s.io/v1alpha1
+kind: OpenStackClusterStackReleaseTemplate
```

Comment:
can you move this file as example file into the root of the repo? So that people can copy it right away and create their own version of it in gitignore?

Reply:
.gitignore is being updated in the PR here https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/pull/88/files